presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -266,6 +281,16 @@ Here are the key components of selector rules in PrestoDB:
 * ``clientTags`` (optional): List of tags. To match, every tag in this list must be in the list of
   client-provided tags associated with the query.
 
+* ``selectorResourceEstimate`` (optional): Resource Group Selection based on resource estimates.
+    - ``executionTime``
+    - ``cpuTime``
+    - ``peakMemory``
```

Comment:
Does ``peakMemory`` relate to ``totalMemoryLimit``? ``executionTimeLimit`` and ``executionTime``, and ``cpuTimeLimit`` and ``cpuTime`` make sense to me, but ``peakMemory`` and ``totalMemoryLimit`` not being the same I found a little confusing. 

Nit: earlier, the items are listed in the order ``executionTimeLimit``, ``totalMemoryLimit``, ``cpuTimeLimit``. 
Suggest ordering the concepts of the two lists in the same order to help the reader, for example ordering this list as ``executionTime``, ``peakMemory``, ``cpuTime``.

Reply:
> Does peakMemory relate to totalMemoryLimit? executionTimeLimit and executionTime, and cpuTimeLimit and cpuTime make sense to me, but peakMemory and totalMemoryLimit not being the same I found a little confusing.

So, they are not directly related to each other. The first section about Resource Group Properties (`executionTimeLimit`, `totalMemoryLimit`, `cpuTimeLimit`) is to set limits for a resource group. This means any query that exceeds these limits will be killed. 

The second section under selector rules (`executionTime`, `peakMemory`, `cpuTime`) is used for determining which resource group a query should be directed to. These are resource estimates that can be passed by the user as a request header(`X-Presto-Resource-Estimate`) or a command line argument(`--resource-estimates`) if the user has an idea of how much resources a query might use.